content/blog/2021/07/webassembly-is-coming-to-cloud-native.md

```diff
@@ -0,0 +1,113 @@
+modules too? We are currently working on a way to run Rego-based policies on
+top of Kubewarden. This will allow operators to have a single policy platform
+to operate. We will talk more about that in a future post, stay tuned for
+updates!
+
+**WebAssembly is secure**, it has been designed with a strong focus on
+security since the beginning. Each WebAssembly module is run inside of a
+dedicated sandbox, with no access neither to other WebAssembly processes nor
+to the host system.
+Kubewarden is a project operating in the security space, that makes even more
+important to us to ensure our policies cannot be turned into an attack vector.  
+Thankfully, WebAssembly and its runtimes already do the heavy lifting for us.
+
+---
+
+So, I kept talking about WebAssembly and how much I'm in love with it. I gave you
+some reasons, but I didn't dig too much into the details. Luckily,
+[The New Stack](https://thenewstack.io/) has recently published not one, but
+**two articles** featuring WebAssembly.  
+These articles do a great job at explaining
+[why everybody keeps talking about WebAssembly](https://thenewstack.io/what-is-webassembly/)
+and [how WebAssembly can even change some computing assumptions](https://thenewstack.io/webassembly-aims-to-eliminate-the-file-system/).  
+Take some time and give them a read, I'm sure you will find them interesting;
+plus, cherry on top, they both feature Kubewarden 😊
+
+
+But we're not over yet, more exciting things happened during this week. The
+following projects started the process to get included into the [CNCF sandbox](https://www.cncf.io/sandbox-projects/):
+[Krustlet](https://krustlet.dev/) (see [here](https://github.com/cncf/toc/issues/690)),
+[wasmCloud](https://wasmcloud.com/) (see [here](https://github.com/cncf/toc/issues/693))
+and
+[ORAS](https://oras.land/) (see [here](https://github.com/cncf/toc/issues/692)).
+
+Let's take a closer look at them!
+
+---
+
+{{< figure class="center" src="/images/krustlet-logo.svg" width="60%" alt="Krustlet logo">}}
+
+[Krustlet](https://krustlet.dev/) registers itself against a Kubernetes clusters
+as a Kubelet instance. However, the Krustlet Kubelet does not run regular containers,
+it instead executes WebAssembly code.
+
+Krustlet leverages [OCI registries](https://opencontainers.org/) to distribute
+the WebAssembly modules to execute. This is exactly the same approach Kubewarden
+adopts to distribute its policies.
+
+As a matter of fact, Krustlet and Kubewarden teams are collaborating on the
+[oci-distribution](https://crates.io/crates/oci-distribution) crate.
+crate. This is the Rust library that Krustlet and Kubewarden use to retrieve
+WasmModules from OCI registries.
+
+---
+
+{{< figure class="center" src="/images/wasmcloud-logo.png" width="60%" alt="wasmCloud logo">}}
+
+[wasmCloud](https://wasmcloud.com/) is a project with a strong focus on the
+developer experience. wasmCloud provides a way to write portable code, code that can be
```

Comment:
```suggestion
developer experience. wasmCloud provides a way to write portable code. Code that can be
```

Reply:
CK is reviewing the doc too. I've seen she has changed this section in a different way. I'll propagate her change